R/package.R

```diff
@@ -33,8 +33,8 @@ use_package <- function(package, type = "Imports", min_version = NULL) {
     refuse_package(package, verboten = c("tidyverse", "tidymodels"))
   }
 
-  use_dependency(package, type, min_version = min_version)
-  how_to_use(package, type)
+  changed <- use_dependency(package, type, min_version = min_version)
+  if (changed) how_to_use(package, type)
```

Comment:
Can you please follow tidyverse style here and use `{}`?

Reply:
Oh, oops, I thought that was just for `return()` and friends! Updating now